test-runner/src/setupTests.js

```diff
@@ -2,4 +2,30 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom';
+import "@testing-library/jest-dom";
+import { setup } from "react-testing-visualizer";
+import path from "path";
+
+setup(path.join(__dirname, "..", "build"));
+
+global.window.document.createRange = () => {
```

Comment:
JSDOM doesn't provide these functions that code mirror uses so we mock them out.